packages/gcf-utils/src/gcf-utils.ts

```diff
@@ -64,17 +64,43 @@ interface EnqueueTaskParams {
 export interface WrapOptions {
   // Whether or not to enqueue direct GitHub webhooks in a Cloud Task
   // queue which provides a retry mechanism. Defaults to `true`.
-  background: boolean;
+  // Deprecated. Please use `maxRetries` and `maxCronRetries` instead.
+  background?: boolean;
 
   // Whether or not to automatically log Octokit requests. Defaults to
   // `false`.
-  logging: boolean;
+  logging?: boolean;
 
   // Whether or not to skip verification of request payloads. Defaults
   // to `true` in test mode, otherwise `false`.
   skipVerification?: boolean;
+
+  // Maximum number of attempts for webhook handlers. Defaults to `10`.
+  maxRetries?: number;
+
+  // Maximum number of attempts for cron handlers. Defaults to `0`.
+  maxCronRetries?: number;
```

Comment:
Little bit worried about the default value of 0.
Do we need to set it properly when migrating? If so, what's the equivalent to the current behavior?

Reply:
Currently, if `background: true`, the cron request is handled directly (so no retries). Otherwise, it's unlimited retries (until the payload expires after a day).

Reply:
Ok